### PR TITLE
[WebServerBundle] fix dependencies

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/WebServerBundle/composer.json [hunk 1]
```diff
@@ -26,6 +27,9 @@
             "/Tests/"
         ]
     },
+    "conflict": {
+        "symfony/dependency-injection": "<3.3"
```
Comment (xabbuh) [inline src/Symfony/Bundle/WebServerBundle/composer.json]: Actually, I do not know why the DI component is no required dependency in our bundles. Is there really a use case for using one of bundles without the DI extension/config?

Comment (stof) [inline src/Symfony/Bundle/WebServerBundle/composer.json]: I think the goal for this bundle is to be able to reuse all the CLI code for Silex too (the DI part here is really just about registering the command services to make them available in Symfony), without creating both a component and a bundle.

Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/composer.json]: Correct

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**